package.json

```diff
@@ -2,8 +2,14 @@
   "name": "react-pg-explain",
   "version": "0.0.1",
   "description": "postgres explain plan visualization",
-  "main": "index.js",
+  "main": "dist/react-pg-explain.cjs.js",
+  "module": "dist/react-pg-explain.esm.js",
+  "browser": "dist/react-pg-explain.umd.js",
```

Comment:
I actually ended up dropping UMD because I couldn't get it to work; I'll take this out. Or do we need to support it? I have only a foggy idea of how the different module systems work.

Reply:
I think we are good to drop this - I'm also not sure whether its needed, but afaik its a legacy system thats no longer in use in modern setups.